_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+</center>
+
+
+But did our initial strategy worked out, and effectively provided us with more resources per dollar spent ?
+
+## Comparing options
+
+One of the biggest unknown impacting pricing on our purchases was hard disks costs, so we decided to focus on cost per TB during our purchases. All our requests were composed of 2 options we believe were, at the time of the purchase, the most cost effective solutions. Comparing for example, 4TB drives versus 6TB drives, 6TB vs 8TB, and so on. We were not comparing just the individual drive costs, but also its impact on the underlying infrastructure. Less storage capacity per server, means more servers, which subsequently means more rack space, more networking capacity, so saving on individual disk costs, could also mean spending more on the infrastructure to support those disks. The challenge being to find the right balance.
+
+## Some numbers
+
+It we look back at our purchases, and focus only on disk and storage server costs (which only gives part of the picture), we get the following table:
+
+| Purchase date  | Disk size  | Cost per Disk  | Cost per disk-TB  | Cost per Server | Server Capacity | Cost per server-TB |
+|---|---|---|---|---|---|---|
+|May 2015       | 4 TB | $270   | $67 | $18,000 | 144 TB | $125 |
+|December 2015  | 6 TB | $380   | $63 | $24,000 | 216 TB | $111 |
+|September 2016 | 8 TB | $550   | $69 | $29,000 | 288 TB | $101 |
+|July 2017      | 10 TB| $620   | $62 | $32,000 | 360 TB | $89  |
+
+___Note: Above prices have been altered but represent a meaningful demonstration___
+
+Although there was a slight spike in September 2016 on individual disk cost, we can see that overall the cost per TB throughout our purchases went down.
+
+## Looking back
+
+Looking back is always easy, but back when we established our procurement strategy, we could only rely on our past experience in the industry, with no guarantees that our strategy relying on purchasing "as late as possible" was going to be effective.
+
+One of the external factors is the USD -> CDN conversion rate, currency fluctuations directly influence our purchase prices. It probably explains why the cost per TB on an individual disk was higher in September 2016. Our storage need at that time was only of 3 servers (864 TB), so the impact was fairly limited and the conversion rate started to go back down after that time and is today at the same level it was early 2015 (although still much higher than when the project started in 2014).
+
+<center>
+  <figure style="width: 70%;">
+      <img src="{{site.urlimg}}francois_gerthoffert/collab-procurement/usd-cdn-conversion-rate.png"/>
+      <figcaption>USD-CDN exchange rate evolution</figcaption>
+  </figure>
+</center>
+
+But ultimately, the thing to look at is that today we are purchasing servers with 2.5 times more storage capacity than in 2015, which means a need for less physical resources per TB of storage, which plays largely in our favor from a budget point of view.
```

Comment:
which means less need for physical resources (racks, switches, cables) per TB of storage,